golang/cosmos/x/vstorage/vstorage_test.go

```diff
@@ -255,4 +255,40 @@ func TestSetWithoutNotify(t *testing.T) {
 
 // TODO: TestValues
 
-// TODO: TestEntries
+func TestEntries(t *testing.T) {
+	kit := makeTestKit()
+	keeper, handler, ctx, cctx := kit.keeper, kit.handler, kit.ctx, kit.cctx
+
+	keeper.SetStorage(ctx, types.NewStorageEntry("key1", "value1"))
+	keeper.SetStorage(ctx, types.NewStorageEntry("key1.child1.grandchild1", "value1grandchild"))
+	keeper.SetStorage(ctx, types.NewStorageEntryWithNoData("key1.child1.grandchild2"))
+	keeper.SetStorage(ctx, types.NewStorageEntry("key1.child1.empty-non-terminal.leaf", ""))
+	keeper.SetStorage(ctx, types.NewStorageEntry("key2.child2.grandchild2", "value2grandchild"))
+	keeper.SetStorage(ctx, types.NewStorageEntry("key2.child2.grandchild2a", "value2grandchilda"))
```

Comment:
Same nit about representative test data

```suggestion
	keeper.SetStorage(ctx, types.NewStorageEntry("key1", "value1"))
	keeper.SetStorage(ctx, types.NewStorageEntry("key1.child1.grandchild1", "value1grandchild"))
	keeper.SetStorage(ctx, types.NewStorageEntryWithNoData("key1.child1"))
	keeper.SetStorage(ctx, types.NewStorageEntry("key1.child1.empty-non-terminal.leaf", ""))
	keeper.SetStorage(ctx, types.NewStorageEntryWithNoData("key2"))
	keeper.SetStorage(ctx, types.NewStorageEntryWithNoData("key2.child2"))
	keeper.SetStorage(ctx, types.NewStorageEntry("key2.child2.grandchild2", "value2grandchild"))
	keeper.SetStorage(ctx, types.NewStorageEntry("key2.child2.grandchild2a", "value2grandchilda"))
```